lib/matplotlib/artist.py

```diff
@@ -1276,7 +1276,9 @@ def properties(self):
                 continue
 
             try:
-                val = func()
+                with warnings.catch_warnings():
+                    warnings.simplefilter('ignore')
```

Comment:
@tacaswell is right. I had to add a similar change in https://github.com/jenshnielsen/matplotlib/commit/0e1be8e5e78b9929c183816309d634ff9ebfabcc to silence a warning in the cycler code